src/scss/components/_modal.scss

```diff
@@ -0,0 +1,37 @@
+.bp-modal {
+  position: fixed;
+  top: 0; left: 0;
+  width: 100%; height: 100%;
+  z-index: 1000;
+
+  &__overlay {
+    width: 100%; height: 100%;
+    top: 0; left: 0;
+    position: absolute;
+    background: rgba(0, 0, 0, 0.7);
+    z-index: 1;
+  }
+
+  &__container {
+    position: relative;
+    z-index: 2;
+    background: #fff;
+    margin: 0 auto 0 auto;
+    max-width: 70%;
+    padding: 2*$m;
+    top: 50%;
+    transform: translateY(-50%);
```

Comment:
Maybe this code for exact center?
transform: translate(-50%, -50%);
top: 50%;
left: 50%;

Reply:
`margin: 0 auto` also works if element has a width defined 😃